renku/core/commands/format/graph.py

```diff
@@ -97,7 +97,6 @@ def dot(graph, simple=True, debug=False, landscape=False, strict=False):
 dot_full = functools.partial(dot, simple=False, landscape=False)
 dot_landscape = functools.partial(dot, simple=True, landscape=True)
 dot_full_landscape = functools.partial(dot, simple=False, landscape=True)
-dot_debug = functools.partial(dot, debug=True)
```

Comment:
why is this removed? I'd prefer if it stays

Reply:
When `renku log --format dot-debug` is invoked, a library uses `cgi.escape()` which is not available in 3.8 anymore and therefore fails. I quickly checked and there is no new version compatible with 3.8 at the moment.
The second commit removing the `dot_debug` was intended to show a possible fix. If it makes sense, I can spend a bit more time trying to understand if `dot_debug` can be kept by using a different library.